dist.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly
+  xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="
+    http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2
+      http://maven.apache.org/xsd/assembly-1.1.2.xsd"
+>
+  <id>dist</id>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <fileSets>
+    <fileSet>
+      <outputDirectory>/bin</outputDirectory>
+      <directory>scripts</directory>
+      <includes>
+          <include>bigdl.sh</include>
+          <include>classes.lst</include>
+          <include>img_class.lst</include>
+      </includes>
+    </fileSet>
+    <fileSet>
+      <outputDirectory>/lib</outputDirectory>
+      <directory>spark/dl/target</directory>
+      <includes>
+          <include>bigdl-*-jar-with-dependencies.jar</include>
```

Comment:
please also add the dependencies-and-spark jar. I notice in some example, some data preprocess job is still standalone java program. We still need that jar